libs/core/src/utils/random-subset.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { randomSubset } from './random-subset';
+
+describe('Core | Utils | random-subset', () => {
+	it('should return a random subset of the given array', () => {
+		const initialArray = [1,2,3,4,5,6,7,8,9];
+		const result = randomSubset(initialArray, 7);
+		expect(initialArray.includes(result[0])).toBeTruthy();
+		expect(initialArray.includes(result[result.length-1])).toBeTruthy();
+	});
```

Comment:
You should make an assertion that each element of the result exists in the original set (definition of subset).

You should also make an assertion that running this twice will likely give two different results.

You could also write a test covering the probability distribution of small case (3 elements) and assert that over a decent number of calls, the distribution of all options is equally likely.

Reply:
I did #1 (before I had tried doing initialArray.forEach I think, and for some reason jasmine doesn't see the expects inside the foreach loop. A regular for loop seems to work though).

#2 I can't really do without doing #3. But that seems kind of extra.

Reply:
Ok, I'm ok with skipping 2 and 3 for now.